content/en/docs/started/installing-kubeflow.md

```diff
@@ -231,14 +238,13 @@ The following table lists <b>legacy distributions</b> which have <b>not had a re
 
 {{% alert title="Warning" color="warning" %}}
 This method is for advanced users.
-
 The Kubeflow community is not able to provide support for environment-specific issues when using the raw manifests.
 If you need support, please consider using a [packaged distribution](#packaged-distributions-of-kubeflow).
 {{% /alert %}}
 
-The raw <a href="https://github.com/kubeflow/manifests">Kubeflow manifests</a> are aggregated by the <a href="https://github.com/kubeflow/community/tree/master/wg-manifests">Manifests Working Group</a> 
-and are intended to be used as the base of packaged distributions,
-<b>advanced users may choose to install the manifests directly</b> by following <a href="https://github.com/kubeflow/manifests#installation">these instructions</a>.
+The raw <a href="https://github.com/kubeflow/manifests"><code>kubeflow/manifests</code></a> are aggregated by the <a href="https://github.com/kubeflow/community/tree/master/wg-manifests">Manifests Working Group</a> 
+and are intended to be used as the <strong>base of packaged distributions</strong>.
+However, advanced users that are willing to make manual patches may choose to install the manifests directly by following <a href="https://github.com/kubeflow/manifests#installation">these instructions</a>.
```

Comment:
What do you mean by manual patches here ?

Reply:
Basically, they need to modify Kustomize overlays manually if they want to have any changes to the default Kubeflow installation. Maybe we should say this rather than manual patches. I am not sure if users will easily understand this message.

Reply:
I am not sure it's super helpful to list out the possible kinds of manual patches, as there are so many.

Do you have a specific wording you think would help people, without confusing them?

Reply:
Maybe we could say:
```
However, advanced users may choose to install the manifests directly by following these instructions. Users can modify the Kubeflow components kustomize manifests to apply required changes before deploying Kubeflow.
```

Reply:
I have pushed some updates, which remove the "manual patches" stuff.

It also links users to the actual release branches of the manifests, rather than `master`, which may or may not actually work.

![Screenshot 2024-02-20 at 12 09 44](https://github.com/kubeflow/website/assets/5735406/f776a55f-ccfe-4a6d-b7cc-d517fdd28e1a)